Replace musicbrainz-vm with musicbrainz-docker (#1526)
metabrainz/musicbrainz-server: INSTALL.md

Installing MusicBrainz Server
=============================

The easiest method of installing a local MusicBrainz Server may be to use the
[MusicBrainz Docker](https://github.com/metabrainz/musicbrainz-docker) Compose

> project, which can be used for web site/service mirror, test, or development.

Reviewer: ```suggestion
project, which can be used as a website/service mirror (including for development).
```
maybe? not sure what is meant by test that isn't encompassed by development

Reviewer 2: "as a website/web service mirror" I guess then?

Reviewer: I thought "website/service" would be understandable enough, but that might be clearer. Mainly just can't bear to see "web site" spelled separately. :P

Author: > maybe? not sure what is meant by test that isn't encompassed by development

Meant to distinguish testing your own app with MB WS/2 (or MB DB), from MBS development.

(The same as in the blog post and the wiki doc.)

Reviewer: Sounds like development too, but ok. :) There are still a couple grammatical issues, but this works too:
```suggestion
project, which can be used for a website/web service mirror, testing, or development.
```

Author: Applied changes in https://github.com/metabrainz/musicbrainz-server/pull/1526/commits/70b89af38d2456888c5fb84d57b860fedc451514, thanks.
